packages/experiments/src/components/FloatingSuggestionsComposite/FloatingSuggestions.tsx

```diff
@@ -0,0 +1,71 @@
+import { IBaseFloatingSuggestionsProps } from './FloatingSuggestions.types';
```

Comment:
If this is going to be promoted from experiments, it's going to need examples... I think the website will pick it up and add it t the list of controls. Could be wrong cc @ecraig12345 to confirm.

Reply:
That's correct, but the examples can be added later. I doubt that we're planning to promote this any time soon.

Reply:
As of now this component will stay in /experiments until the UnifiedPicker is completely functional with tests and example.
Promoting this as fundamental component is a long shot.